content/en/docs/tasks/security/authentication/mtls-migration/index.md

```diff
@@ -120,179 +61,87 @@ the policies to enforce STRICT mutual TLS between the workloads.
 * Verify setup by sending an http request (using curl command) from any sleep pod (among those in namespace `foo`, `bar` or `legacy`) to `httpbin.foo`.  All requests should success with HTTP code 200.
 
     {{< text bash >}}
-    $ for from in "foo" "bar" "legacy"; do kubectl exec $(kubectl get pod -l app=sleep -n ${from} -o jsonpath={.items..metadata.name}) -c sleep -n ${from} -- curl http://httpbin.foo:8000/ip -s -o /dev/null -w "sleep.${from} to httpbin.foo: %{http_code}\n"; done
+    $ for from in "foo" "bar" "legacy"; do for to in "foo" "bar"; do kubectl exec $(kubectl get pod -l app=sleep -n ${from} -o jsonpath={.items..metadata.name}) -c sleep -n ${from} -- curl http://httpbin.${to}:8000/ip -s -o /dev/null -w "sleep.${from} to httpbin.${to}: %{http_code}\n"; done; done
     sleep.foo to httpbin.foo: 200
+    sleep.foo to httpbin.bar: 200
     sleep.bar to httpbin.foo: 200
+    sleep.bar to httpbin.bar: 200
     sleep.legacy to httpbin.foo: 200
+    sleep.legacy to httpbin.bar: 200
     {{< /text >}}
 
 * Also verify that there are no authentication policies or destination rules (except control plane's) in the system:
 
     {{< text bash >}}
-    $ kubectl get policies.authentication.istio.io --all-namespaces
-    NAMESPACE      NAME                          AGE
-    istio-system   grafana-ports-mtls-disabled   3m
+    $ kubectl get peerauthentication --all-namespaces
+    No resources found
     {{< /text >}}
 
     {{< text bash >}}
     $ kubectl get destinationrule --all-namespaces
-    NAMESPACE      NAME                                 HOST                                             AGE
-    istio-system   istio-multicluster-destinationrule   *.global                                         35s
-    istio-system   istio-policy                         istio-policy.istio-system.svc.cluster.local      35s
-    istio-system   istio-telemetry                      istio-telemetry.istio-system.svc.cluster.local   33s
+    No resources found
     {{< /text >}}
 
-### Configure clients to send mutual TLS traffic
+## Lock down to mutual TLS
```

Comment:
Per other discussion, Samrat suggests not using this phrase *lock down*. *Stricly require mutual TLS*, may be?

Also, mention namespace to avoid confusion with the next section. Though I think we can also collapsed into one (just show at mesh or namespace level and leave the other as exercise)

Reply:
referred the ns to avoid confusion.

i'll defer terminology swap to future iteration. whoever feel strong about it can drive it. we get the doc accurate to impl first.